packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -226,10 +227,13 @@ export const GridRootStyles = styled('div', {
     [`& .${gridClasses.cell}`]: {
       display: 'flex',
       alignItems: 'center',
+      overflow: 'hidden',
+      borderBottom: `1px solid ${borderColor}`,
```

Comment:
> In that case, the text won't overflow but break and fit into the cell, which actually should be the expected result but it opened for debates.

Removing `whiteSpace: nowrap` broke our demos. If it was an alpha version we could say that it's expected, but doing it now is risky.

![image](https://user-images.githubusercontent.com/42154031/154075177-6d2653c3-27e9-46cb-b2b4-53bb5b4a97a7.png)

Reply:
Yeah ... ok the fix the easy part is that it is fixable by just keeping the `textOverflow: 'ellipsis'` and `whiteSpace: 'nowrap'` on the cell class.

Reply:
`textOverflow` in the cell does nothing, but `whiteSpace` should be kept.